### DISCUSSION TITLE
duda con 'cargar_arreglo'

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#90] [Preguntas y respuestas] [answered]
Hola! me surgio una duda con el ejercicio 1.1 del tp3. cuando quiero llamar a la funcion para cargar los arreglos no es necesario que almacene los valores en una variable (Ej: `int arreglo_1 = cargar_arreglo(capacidad_1, numeros_1);`)?

**Comment by martinvilu** [comment, marked as answer]
¿Que es lo que estas retornando?

**Reply by rinrot** [reply]
me acabo de dar cuenta q no tengo que retornar nada porq los cambios d los arrays se guardan fuera de la funcion. m pasa por no dar bola en clase

**Reply by martinvilu** [reply]
En la consigna hay una pista a _que podes_ retornar en las funciones de este TP.

**Comment by martinvilu** [comment]
> me pasa por no dar bola en clase


![imagen_725922](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/6f4f62a0-d386-42ab-b565-d6c4c24f9ad6)

**Comment by martinvilu** [comment]
En este código:
```C
int arreglo_1 = cargar_arreglo(capacidad_1, numeros_1);
```
Algo para ver en esto, es que el tipo de `arreglo_1`, no 'alcanza' para guardar el arreglo en sí, dicho rápidamente, no están los `[]`

Planteado como:
```C
int arreglo_1[] = cargar_arreglo(capacidad_1, numeros_1);
```
_funcionaria_, pero hay un problema con la creación del arreglo que vamos a ver el miércoles próximo. (una cuestión **muy** interesante).

**Reply by rinrot** [reply]
Gracias por responder profe! por ahora voy guardando el cambio del arreglo con una funcion void. espero que el uso de ese tipo de retorno sea valido para la resolucion del ejercicio.
